extensions/blocks/changelog/components/time-stamp-control.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	BaseControl,
+	__experimentalNumberControl as NumberControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+function checkValidValue( val, max ) {
+	if ( val > max ) {
+		return val = max;
+	} else if ( val < 0 ) {
+		return val = 0;
```

Comment:
`=` or `==`? Also 2 lines above

Reply:
it should be just returned `0`. Updating. Thanks :-)

Reply:
Updated here https://github.com/Automattic/jetpack/pull/17885/commits/035c9626624f0076793abb393626b55669190009